rollup.config.mjs

```diff
@@ -73,3 +75,27 @@ export default {
     }),
   ],
 };
+
+/**
+ * If env var `THIRD_PARTY_NOTICES_REVISION` is set to `true`, fetches the
+ * revision of a dependency from the npm registry. Falls back to `N/A` if the
+ * revision cannot be fetched for any reason. This is useful for chromedriver
+ * build, as the specific dependency revisions are required for the build.
+ */
+function getRevision(dependency) {
+  if (process.env.THIRD_PARTY_NOTICES_REVISION !== 'true') return 'N/A';
+  try {
+    // Chromium requires the revision to be specified for any dependency.
+    // Even though the revision is not available in the local npm package info,
+    // it can be fetched from the registry.
+    const npmRegistryUrl = `https://registry.npmjs.org/${dependency.name}/${dependency.version}`;
+    const dependencyRegistryInfo = JSON.parse(
+      execSync(`curl -s ${npmRegistryUrl}`).toString()
```

Comment:
let's use spawnSync with args so that there is no risk that package name is interpreted by the shell

Reply:
NP, but this depends on the previous discussion.